histomicstk/segmentationschool/Codes/IterativePredict_1X.py

```diff
@@ -205,8 +202,11 @@ def predict(args):
             print(basename)
             # print(extname)
             # try:
-            slide=openslide.TiffSlide(wsi)
-            print(wsi,'here/s the silde')
+            try:
+                slide=openslide.TiffSlide(wsi)
```

Comment:
You can do openslide.OpenSlide(wsi') as tiffslide is a drop-in replacement for openslide and tiffslide has been imported as openslide.